packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -731,52 +761,70 @@ export class StatsAnalyzer extends EventsScope {
         false
       );
 
+      const currentTotalPacketsSent = getCurrentStatsTotals('video-send', 'totalPacketsSent', true);
+      const previousTotalPacketsSent = getPreviousStatsTotals(
+        'video-send',
+        'totalPacketsSent',
+        true
+      );
+
+      const currentFramesEncoded = getCurrentStatsTotals('video-send', 'framesEncoded', true);
+      const previousFramesEncoded = getPreviousStatsTotals('video-send', 'framesEncoded', true);
+
+      const currentFramesSent = getCurrentStatsTotals('video-send', 'framesSent', true);
+      const previousFramesSent = getPreviousStatsTotals('video-send', 'framesSent', true);
+
+      let doesVideoSendExist = false;
+      Object.keys(this.lastStatsResults).forEach((item) => {
+        if (item.includes('video-send')) {
+          doesVideoSendExist = true;
```

Comment:
Also here. In this case, using `Array.some()` might make more sense.

Reply:
Updated.